tests/ha/test_pod_failure.py

```diff
@@ -148,7 +148,10 @@ def teardown_method(self):
         LOGGER.info("STARTED: Teardown Operations.")
         if self.s3_clean:
             LOGGER.info("Cleanup: Cleaning created IAM users and buckets.")
-            resp = self.ha_obj.delete_s3_acc_buckets_objects(self.s3_clean)
+            if HA_CFG["dtm0_disabled"]:
```

Comment:
use get function while accessing key of a dict. Keep dtm disabled by default.

Reply:
this is a boolean flag. Ideally should be part of cmn_cfg/ha_cfg through the setup specific DB config.

Reply:
@seagate-sarang-sawant not able to understand comment here, so I'm reading this from ha_config.yaml and the default value is set to true for now and once we have DTM0 enabled, will set it to false so we don't need anymore chnages in test files.

Reply:
@seagate-sarang-sawant as per discussion shifted the value to common config and changed in test file accordingly. Also default value is dtm enabled as by the time we start validation it would be enabled is what I believe.